### PR TITLE
Increase piksi_firmware init UART timeout past watchdog

### PR DESCRIPTION
**Pull request by benjaminaltieri**
Minimal bench testing here, but I have added logging facility to allow these activities to be caught in `syslog` while increasing the timeout on waiting for uarts to come up.

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.19-3-g8c65f440
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=628844)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-3-g8c65f440&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-3-g8c65f440&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.19-3-g8c65f440
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.19-3-g8c65f440)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.19-3-g8c65f440/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.19-3-g8c65f440

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.19-3-g8c65f440
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=628848)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-3-g8c65f440&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-3-g8c65f440&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.19-3-g8c65f440
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.19-3-g8c65f440)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.19-3-g8c65f440/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.19-3-g8c65f440

File: board/piksiv3/rootfs-overlay/etc/init.d/S85piksi_firmware [hunk 1]
```diff
@@ -2,10 +2,17 @@
...
+log_tag=$name # required by logging.sh
+source /etc/init.d/logging.sh
+
+setup_loggers
+
 settings_ini=/persistent/config.ini
 
-uart_wait_count_limit=20 # wait 2s
-uart_check_interval=0.1
+uart_wait_count_limit=200 # wait 200s
```
Comment (silverjam) [inline board/piksiv3/rootfs-overlay/etc/init.d/S85piksi_firmware]: Let's put the interval back to 0.1 and the count to 2000, this way in the normal case we won't be adding a second to the boot time if we happen to miss the startup of the adapter.

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.19-4-g58c33861
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=628901)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-4-g58c33861&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-4-g58c33861&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.19-4-g58c33861
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.19-4-g58c33861)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.19-4-g58c33861/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.19-4-g58c33861

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.19-4-g58c33861
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=628907)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-4-g58c33861&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.19-4-g58c33861&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.19-4-g58c33861
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.19-4-g58c33861)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.19-4-g58c33861/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.19-4-g58c33861

**Comment by benjaminaltieri** [comment]
backport should be rolled in with #728

**Pull request merged**